aisdc/attacks/attack.py

```diff
@@ -14,3 +17,18 @@ def attack(self, dataset: Data, target_model: sklearn.base.BaseEstimator) -> Non
 
     def __str__(self):
         raise NotImplementedError
+
+
+class ConfigFile:  # pylint: disable=too-few-public-methods
+    """ConfigFile class to load parameters from json configuration file"""
+
+    def __init__(self, config_filename):
+        self.config_filename = config_filename
+
+    def load_config_file_into_dict(self, attack_args_dict: dict) -> None:
+        """Reads a configuration file and loads it into a dictionary object"""
+        if os.path.isfile(self.config_filename):
```

Comment:
One final change here -- please add something in an `else` for the `if os.path...` to raise a log error. Or, just remove the check and use the fact that python will raise a FileNotFoundException anyway...(I prefer the latter)

Reply:
Thanks @simonrnss. I have removed the if statement (if os.path...) as you suggested.